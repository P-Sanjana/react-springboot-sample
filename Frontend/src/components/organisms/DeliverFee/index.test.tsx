import React from 'react';
import { render } from '@testing-library/react';
import DeliveryComponent from './index';

describe('Checkout Component', () => {
  it('should match snapshot', () => {
    const wrapper = render(
      <DeliveryComponent />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
