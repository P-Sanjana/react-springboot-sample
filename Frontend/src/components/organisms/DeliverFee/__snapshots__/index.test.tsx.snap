// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkout Component should match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="MuiPaper-root MuiCard-root makeStyles-parentCard-6 MuiPaper-outlined MuiPaper-rounded"
      >
        <p
          class="MuiTypography-root MuiTypography-body1"
          style="color: rgb(52, 52, 70); font-weight: bold; margin: 8px;"
        >
          Select speed delivery
        </p>
        <div
          class="MuiGrid-root makeStyles-cards-8 MuiGrid-container MuiGrid-align-items-xs-center MuiGrid-justify-content-xs-space-between"
        >
          <div
            class="MuiGrid-root makeStyles-content-5 MuiGrid-item"
          >
            <div
              class="MuiBox-root MuiBox-root-10 makeStyles-imageBox-2"
            >
              <img
                alt="currency image"
                class="makeStyles-currencyImage-4"
                src="truck-line.svg"
              />
            </div>
            <div
              class="MuiBox-root MuiBox-root-11 makeStyles-currency-1"
            >
              <p
                class="MuiTypography-root MuiTypography-body1"
                style="color: rgb(52, 52, 70); font-weight: bold;"
              >
                Instant : 2-5 min
              </p>
              <span
                class="MuiTypography-root makeStyles-currencyCode-3 MuiTypography-caption"
              >
                Transaction fees : 0.001 BTC
              </span>
            </div>
          </div>
          <div
            class="MuiGrid-root makeStyles-arrow-9 MuiGrid-item"
          >
            <img
              src="chevron-down.svg"
            />
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="MuiPaper-root MuiCard-root makeStyles-parentCard-6 MuiPaper-outlined MuiPaper-rounded"
    >
      <p
        class="MuiTypography-root MuiTypography-body1"
        style="color: rgb(52, 52, 70); font-weight: bold; margin: 8px;"
      >
        Select speed delivery
      </p>
      <div
        class="MuiGrid-root makeStyles-cards-8 MuiGrid-container MuiGrid-align-items-xs-center MuiGrid-justify-content-xs-space-between"
      >
        <div
          class="MuiGrid-root makeStyles-content-5 MuiGrid-item"
        >
          <div
            class="MuiBox-root MuiBox-root-10 makeStyles-imageBox-2"
          >
            <img
              alt="currency image"
              class="makeStyles-currencyImage-4"
              src="truck-line.svg"
            />
          </div>
          <div
            class="MuiBox-root MuiBox-root-11 makeStyles-currency-1"
          >
            <p
              class="MuiTypography-root MuiTypography-body1"
              style="color: rgb(52, 52, 70); font-weight: bold;"
            >
              Instant : 2-5 min
            </p>
            <span
              class="MuiTypography-root makeStyles-currencyCode-3 MuiTypography-caption"
            >
              Transaction fees : 0.001 BTC
            </span>
          </div>
        </div>
        <div
          class="MuiGrid-root makeStyles-arrow-9 MuiGrid-item"
        >
          <img
            src="chevron-down.svg"
          />
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
